### PR TITLE
Issue #80 - Converted volunteers page to use templates

### PR DESCRIPTION
**Pull request by EChesters**
- Removed head, footer, and social
- Added newsletter partial and jekyll info like page title
- Realigned HTML + removed spaces in between tags
- Removed a stray end div tag

### CODE REVIEW

File: src/volunteers.html [hunk 1]
```diff
@@ -1,253 +1,84 @@
...
+    <a href="volunteer-contact.html" class="button">Volunteer with us</a>
+  </div>
+</section>
+<section>
+  <div class="content">
+    <h2 class="section-heading">How it works</h2>
+    <div class="col-wrap">
+      <div class="col-3 centered-text">
+        <div class="">
+          <img class="non-profit-icons" src="img/icons/email-us.svg" alt="Email Us Image"></a>
```
Comment (tanyapowell) [inline src/volunteers.html]: where does this `</a>` open?

File: src/volunteers.html [hunk 2]
```diff
@@ -1,253 +1,84 @@
...
-          <div class="col-3">
-            <div class="">
-              <img class="non-profit-icons" src="img/icons/project-starts.svg" alt="Email Confirmation Image"></a>
-            </div>
-            <h2 class="section-heading">3. Done!</h2>
-            <p>Choose your project and get hacking!</p>
-          </div>
+      <div class="col-3">
+        <div class="">
+          <img class="non-profit-icons" src="img/icons/email-confirmation.svg" alt="Email Confirmation Image"></a>
```
Comment (tanyapowell) [inline src/volunteers.html]: Could you double what's going on with the `` closing tags? They're all `</a>`

File: src/volunteers.html [hunk 3]
```diff
@@ -1,253 +1,84 @@
...
-            </ul>
-          </div>
-          <div class="col-xs-12">
-            <div class="col-xs-12 col-md-7 position central text-center">
-              <p>Copyright &#xA9; Women Hack For Non-Profits 2016.</p>
-            </div>
-          </div>
+      <div class="col-3">
+        <div class="">
+          <img class="non-profit-icons" src="img/icons/project-starts.svg" alt="Email Confirmation Image"></a>
```
Comment (tanyapowell) [inline src/volunteers.html]: Conflicting tags

File: src/volunteers.html [hunk 4]
```diff
@@ -1,253 +1,78 @@
...
-    </script>
-  </body>
-</html>
\ No newline at end of file
+    </div>
+  </div>
+</section>
+<section>
+  <h2 class="section-heading">Meet our volunteers</h2>
+  <p>Meet some of our volunteers to find out what you could be both contributing and learning by joining WHFNP.<p>
```
Comment (lili2311) [inline src/volunteers.html]: Could you please close the `p` here?

**Comment by tanyapowell** [comment]
👍

**Pull request merged**